internals/models.py

```diff
@@ -1740,6 +1740,36 @@ def remove_as_component_owner(self, component_name):
         component_name, remove_as_owner=True)
 
 
+class APIOwner(DictModel):
+  """Describes the properties to store raw API_OWNERS content."""
+  url = ndb.StringProperty()
```

Comment:
Maybe make url and raw_content required?
I generally don't make a lot of fields required, but in this case the URL is like the key, and the raw_content is pretty much the point of the whole entity.